src/demo/pages/PanePage/examples/Pane.Basic.Example.tsx

```diff
@@ -0,0 +1,113 @@
+                    </WrappedContent>
+                    <PaneContent className='ms-font-m'>
+                        Pane content goes here.
+                    </PaneContent>
+                </Pane>
+            </div>
+        );
+    }
+
+    @autobind
+    private _showPane() {
+        this.setState((prevState: IPaneSmallRightExampleState) => {
```

Comment:
I think there is something I'm not understanding in VS Code.  Some files it auto-formats to 2 space and some to 4 space.  I see the workspace settings are set to 2 space but I haven't figured out why that isn't being followed.  Auto format on save is also turned on so this stuff just happens without warning.